### ISSUE TITLE
Crashing on division by zero

### ISSUE THREAD

**Issue by ggirotto** [patchthecode/JTAppleCalendar#775]
Library Version: 7.1.5

My app is crashing due to a division by zero when trying to reload an array of dates.

I'm using the function `reloadDates` and i'm sending two equal dates `2018-05-02 22:36:03 UTC` and the lib crashes inside the function 
```swift
indexPathOfdateCellCounterPath(_ date: Date, dateOwner: DateOwner) -> IndexPath?
``` 
when trying to perform a division by zero.

I've attached some images to better illustrate the situation.

![screen shot 2018-05-03 at 1 40 47 pm](https://user-images.githubusercontent.com/11745745/39592501-536e2a58-4edd-11e8-8791-5511d604f898.png)
![screen shot 2018-05-03 at 2 21 47 pm](https://user-images.githubusercontent.com/11745745/39592502-538fd0d6-4edd-11e8-916f-d8b691059fd4.png)

**Comment by ggirotto** [comment]
@patchthecode I've investigated a little further and found what was causing the problem.
I was fetching the data that I will display in the calendar and trying to reload the dates that had some content to show in `viewWillAppear` function. But for some reason, at this point the calendar didn't loaded itself yet, because I've printed the`numberOfItemsInSection ` and was returning 0 for all sections, that's why the 'division by zero' error. I've moved this function the fetch the necessary data to `viewDidAppear` function and all worked flawless.
